openedx_webhooks/tasks/github.py

```diff
@@ -1,30 +1,34 @@
 # coding=utf-8
 from __future__ import unicode_literals, print_function
 
+from datetime import date
 import json
 import re
-from datetime import date
 
-from iso8601 import parse_date
 from flask import render_template, render_template_string
+from iso8601 import parse_date
 from urlobject import URLObject
+import requests
```

Comment:
I usually sort `from` imports before `import`. Is it convention here to just sort by package name? Actually most code I've seen doesn't sort at all.